### PR TITLE
Publish some LuaC API

### PR DESCRIPTION
**Pull request by numberZero**

### CODE REVIEW

File: mesecons_luacontroller/init.lua [hunk 1]
```diff
@@ -522,6 +522,22 @@ local digiline = {
...
+
+local function set_program(pos, code)
+	reset(pos)
+	reset_meta(pos, code)
+	local err = run(pos, {type="program"})
+	if err then
+		print(err)
+		reset_meta(pos, code, err)
+	end
+end
```
Comment (numberZero) [inline mesecons_luacontroller/init.lua]: Should return whether the update was successful... or not?

Comment (DS-Minetest) [reply mesecons_luacontroller/init.lua]: Or return `err`. Or whether the update was successful... or not `, err`.

File: mesecons_luacontroller/init.lua [hunk 2]
```diff
@@ -522,6 +522,23 @@ local digiline = {
...
+
+local function set_program(pos, code)
+	reset(pos)
+	reset_meta(pos, code)
+	local err = run(pos, {type="program"})
+	if err then
+		reset_meta(pos, code, err)
+		return false, err
+	end
+	return true
```
Comment (DS-Minetest) [inline mesecons_luacontroller/init.lua]: You could save one indention with
```lua
if not err then
	return true
end
```

**Comment by DS-Minetest** [comment]
Btw does this close #346?

**Comment by numberZero** [comment]
@DS-Minetest no ofc, but it makes #346 easier to do properly.

**Pull request merged**